Marlin/src/module/planner.cpp

```diff
@@ -2419,7 +2563,7 @@ bool Planner::_populate_block(block_t * const block, bool split_move,
     accel = CEIL((esteps ? settings.acceleration : settings.travel_acceleration) * steps_per_mm);
 
     #if ENABLED(LIN_ADVANCE)
-
+      // TODO (DerAndere): Add support for LINEAR_AXES >= 4
```

Comment:
Another TODO...

Reply:
LIN_ADVANCE was never compatible with LINEAR_AXES >= 4, and it would be probably very difficult to implement. that is why I added a sanity check.

Reply:
That’s fine, I didn’t notice the sanity check. Personally, I don’t think we should be leaving TODO comments. A normal comment saying incompatible with > 3 axis would be fine, with nothing implying anybody is returning to add it.